state/store_test.go

```diff
@@ -173,6 +173,7 @@ func TestPruneStates(t *testing.T) {
 						{Data: []byte{2}},
 						{Data: []byte{3}},
 					},
+					AppHash: make([]byte, 1),
```

Comment:
The same change should be added to `TestABCIResPruningStandalone` and `TestFinalizeBlockResponsePruning` as well.

Reply:
Thanks, did it in 1a0d470883